heartbeat/docs/configuring-howto.asciidoc

```diff
@@ -1,8 +1,12 @@
 [[configuring-howto-heartbeat]]
-= Configuring {beatname_uc}
+= {beatname_uc} configuration
```

Comment:
I'm curious as to why you went with "`{beatname_uc} configuration`" over "`Configure {beatname_uc}`"? All other headings follow the latter pattern.

Reply:
Because this section is meant to be reference content and verbs/gerunds are generally reserved for task-based content. However...I think it does look a little weird, so I'm going to change it back. :-)

Reply:
Whichever you prefer is fine with me. Just trying to learn.